packages/deployer/src/services/deps.ts

```diff
@@ -49,18 +49,18 @@ export class Deps extends MastraBase {
     }
   }
 
-  public async install({ dir = this.rootDir, packages = [] }: { dir?: string; packages?: string[] }) {
+  public async install({ dir = this.rootDir }: { dir?: string }) {
     let runCommand = this.packageManager;
 
     switch (this.packageManager) {
-      case 'npm':
-        runCommand = `${this.packageManager} i`;
-        break;
       case 'pnpm':
         runCommand = `${this.packageManager} --ignore-workspace install`;
         break;
+      case 'yarn':
+        // similar to --ignore-workspace but for yarn
+        await ensureFile(path.join(dir, 'yarn.lock'));
```

Comment:
curious, why do we need to write it? wont yarn do this?

Reply:
there is a yarn.lock in the root, if we do yarn in a non-workspace folder, it crashes. Creating a new yarn.lock file, creates a new yarn root.